src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue22549.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue22549 : _IssuesUITest
+	{
+		public Issue22549(TestDevice device) : base(device) { }
+
+		public override string Issue => "Binding Border.StrokeShape not working";
+
+		[Test]
+		public void BindingOnStrokeShapeShouldWork()
+		{
+			App.WaitForElement("button");
+
+			//Border should have radius
+			VerifyScreenshot();
+
+			App.Click("button");
+
+			//The test passes if border radius is equal to 0
+			VerifyScreenshot();
```

Comment:
These may need to have a custom filename since they are different.

Reply:
Right, added different names.